### DISCUSSION TITLE
Stripe format - contiguous or not

### DISCUSSION THREAD

**Discussion by david3a** [vsf-tv/gccg-api#35] [General]
![image](https://github.com/user-attachments/assets/6713a041-3c44-45f7-a906-16f21ef22554)

**Comment by david3a** [comment]
We have implemented stripe processing now, we have restricted to only 8 stripes and supporting video only, using the "segemented" flag in the payload json to signal that we will uses slices, is the expected use fo the flag, is so should we call it "slices" or "sliced" to keep consistent?

Worth noting that each slice is effectively a mini frame, you cannot easily translate between slices and a single frame plane without a copy operation

![GCCG Stripe working drawio](https://github.com/user-attachments/assets/b33d0fd7-a5d3-4fa7-bd20-8fd7ffb85ba4)

**Reply by david3a** [reply]
this could be managed by using a stride size to allow the suggestion of Y samples then U the V and optionally Alpha to be sequential and then use a stride to allow stepping between lines.

with 422 1920 stride is 3840 
![GCCG Stride managed Y U V for slice working with planes drawio](https://github.com/user-attachments/assets/cb154caf-cffe-49b2-9df6-197b68bba064)

**Reply by kierank** [reply]
If you do this, decide whether stride is in pixels or bytes please.

**Reply by david3a** [reply]
agreed! i think there are arguments both ways!
